mmv1/products/cgc/terraform.yaml

```diff
@@ -472,6 +472,32 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         ignore_read_extra:
           - "deletion_protection"
           - "root_password"
+    ### Password Validation Policy ###
+      - !ruby/object:Provider::Terraform::Examples
+        name: "sql_mysql_instance_pvp"
+        primary_resource_id: "mysql_pvp_instance_name"
+        primary_resource_type: "google_sql_database_instance"
+        vars:
+          mysql_pvp_instance_name: "mysql-pvp-instance-name"
+          deletion_protection: "true"
+        test_vars_overrides:
+          deletion_protection: "false"
+        ignore_read_extra:
+          - "deletion_protection"
+          - "root_password"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "sql_postgres_instance_pvp"
+        primary_resource_id: "postgres_pvp_instance_name"
+        primary_resource_type: "google_sql_database_instance"
+        vars:
+          postgres_pvp_instance_name: "postgres-pvp-instance-name"
+          deletion_protection: "true"
+        skip_test: true
+        test_vars_overrides:
+          deletion_protection: "false"
+        ignore_read_extra:
+          - "deletion_protection"
+          - "root_password"
```

Comment:
Sorry for missing this in my last review. 
Product `cgc` is introduced to generated CGC (cloud.google.com) Samples from Magic Modules. Sql database instance feature test are added in the resource test file, which is `mmv1/third_party/terraform/tests/resource_sql_database_instance_test.go.erb` file.
You can simply copy the test generated in the downstream ([here](https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-6329-old..auto-pr-6329#diff-7e0d44a3667b09addcfbba7a5d187db5489c7eec6db9c3681b721b0bd1375269R1037)) and add it to `mmv1/third_party/terraform/tests/resource_sql_database_instance_test.go.erb` file, and also change the test name to have prefix `TestAccSqlDatabaseInstance_` instead of `TestAccCGCSnippet_`.